addons/survey/views/survey_templates_statistics.xml

```diff
@@ -189,38 +192,54 @@
     </template>
 
     <template id="question_result_text" name="Question: text result (text_box, char_box)">
-        <table class="table table-hover table-sm" t-att-id="'survey_table_question_%d' % question.id">
-            <thead>
-                <tr>
-                    <th>#</th>
-                    <th>User Responses</th>
-                </tr>
-            </thead>
-            <tbody>
-                <t t-foreach="table_data" t-as="input_line">
-                    <tr t-if="not input_line.skipped" t-att-class="'d-none' if input_line_index >= page_record_limit else ''">
-                        <td>
-                            <t t-if="no_print_url"><t t-esc="input_line_index + 1"></t></t>
-                            <t t-else="">
-                                <a t-att-href="input_line.user_input_id.get_print_url()">
-                                    <t t-esc="input_line_index + 1"></t>
-                                </a>
-                            </t>
-                        </td>
-                        <td>
-                            <t t-if="question.question_type == 'text_box'">
-                                <span t-if="no_print_url" t-field="input_line.value_text_box" />
-                                <a t-else="" t-att-href="input_line.user_input_id.get_print_url()" t-field="input_line.value_text_box"></a>
-                            </t>
-                            <t t-if="question.question_type == 'char_box'">
-                                <span t-if="no_print_url" t-field="input_line.value_char_box" />
-                                <a t-else="" t-att-href="input_line.user_input_id.get_print_url()" t-field="input_line.value_char_box"></a>
-                            </t>
-                        </td>
+        <div t-att-id="'survey_page_question_%d' % question.id">
```

Comment:
This looks like it adds an additional wrapper, which makes the diff a lot bigger, and I'm not sure why exactly?
The ID does not seem used?
Maybe I missed it.

Reply:
Mmmh, then this diff is probably in the wrong commit? Do we still need it following our new approach?